contracts/utils/test/artifacts.ts

```diff
@@ -6,6 +6,7 @@
 import { ContractArtifact } from 'ethereum-types';
 
 import * as Authorizable from '../test/generated-artifacts/Authorizable.json';
+import * as D18 from '../test/generated-artifacts/D18.json';
```

Comment:
D18? I must be missing some context here

Reply:
Just yet another fixed point library to jive with dydx.

Reply:
Fun 🎉